CONTRIBUTING.md

```diff
@@ -77,6 +77,42 @@ import (
 
 ```
 
+### Instrumentation
+
+- **Metrics**: Tempo is instrumented with [Prometheus metrics](https://prometheus.io/). It emits RED metrics for most
+  services and backends. The relevant dashboards can be found in the [Tempo mixin](operations/tempo-mixin).
+- **Logs**: Tempo uses the [go-kit level logging library](https://pkg.go.dev/github.com/go-kit/kit/log/level) and emits
+  logs in the `key=value` (logfmt) format.
+- **Traces**: Tempo uses the [Jaeger Golang SDK](https://github.com/jaegertracing/jaeger-client-go) for tracing instrumentation.
+  As of this writing, only the read path of tempo is instrumented for tracing.
+
+### Testing
```

Comment:
Should we mention also `docker-compose`, `helm` and `tk` as testing too?

Reply:
Yeah that's a good point. I'll add something in here.